lib/plugins/object.js

```diff
@@ -434,22 +437,27 @@ module.exports = fp(
       /**
        * Put an Object variant into the Ledger
        *
-       * @param {Object} opts Method parameters
-       * @param {string} opts.name Object name
-       * @param {string} opts.env Environment
-       * @param {string} [opts.variant] Object variant name. Default `_default`
-       * @param {string} opts.expiration Optional variant expiration. Default `never`
-       * @param {any} opts.data Object variant data
+       * @param {Object} params Method parameters
+       * @param {string} params.name Object name
+       * @param {string} params.env Environment
+       * @param {string} [params.variant] Object variant name. Default `_default`
+       * @param {string} params.expiration Optional variant expiration. Default `never`
+       * @param {any} params.data Object variant data
+       * @param {Object} opts Method options
+       * @param {boolean} [opts.forceCreateEnv] Force to create enviroment
        * @returns {Promise<any>} The operation result
        */
-      async function putObjectVariant({
-        name: keyname,
-        version,
-        env,
-        variant = '_default',
-        expiration = null,
-        data
-      }) {
+      async function putObjectVariant(
+        {
+          name: keyname,
+          version,
+          env,
+          variant = '_default',
+          expiration = null,
+          data
+        },
+        { forceCreateEnv = false } = { forceCreateEnv: false }
```

Comment:
Minor, but could be:

```suggestion
        { forceCreateEnv = false } = { }
```